src/main/python/api/house_service.py

```diff
@@ -0,0 +1,14 @@
+import flask
+from flask import jsonify
+
+from src.main.python.datastore import house_dao
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+
+@app.route('/', methods=['GET'])
+def fetch_all_houses():
+    return jsonify(house_dao.get_all_houses()), 200
```

Comment:
So this is giving me ALL 27 million houses... Probably a stupid question but do we need all houses. Like does the data need to be cleaned at all? Do we have duplicates we can remove?